layouts/shortcodes/observability_pipelines/processors/sds_custom_rules.md

```diff
@@ -0,0 +1,41 @@
+    - Backreferences and capturing sub-expressions (lookarounds)
+    - Arbitrary zero-width assertions
+    - Subroutine references and recursive patterns
+    - Conditional patterns
+    - Backtracking control verbs
+    - The `\C` "single-byte" directive (which breaks UTF-8 sequences)
+    - The `\R` newline match
+    - The `\K` start of match reset directive
+    - Callouts and embedded code
+    - Atomic grouping and possessive quantifiers
+1. For **Create keyword dictionary**, add keywords to refine detection accuracy when matching regex conditions. For example, if you are scanning for a sixteen-digit Visa credit card number, you can add keywords like `visa`, `credit`, and `card`. You can also require that these keywords be within a specified number of characters of a match. By default, keywords must be within 30 characters before a matched value.
+1. In the **Define rule target and action** select if you want to scan the **Entire Event**, **Specific Attributes**, or **Exclude Attributes** in the dropdown menu.
+    - If you are scanning the entire event, you can optionally exclude specific attributes from getting scanned. Use [path notation](#path-notation-example-custom) (`outer_key.inner_key`) to access nested keys. For specified attributes with nested data, all nested data is excluded.
+    - If you are scanning specific attributes, specify which attributes you want to scan. Use [path notation](#path-notation-example-custom) (`outer_key.inner_key`) to access nested keys. For specified attributes with nested data, all nested data is scanned.
+1. For **Define actions on match**, select the action you want to take for the matched information. **Note**: Redaction, partial redaction, and hashing are all irreversible actions.
+    - **Redact**: Replaces all matching values with the text you specify in the **Replacement text** field.
+    - **Partially Redact**: Replaces a specified portion of all matched data. In the **Redact** section, specify the number of characters you want to redact and which part of the matched data to redact.
+    - **Hash**: Replaces all matched data with a unique identifier. The UTF-8 bytes of the match is hashed with the 64-bit fingerprint of FarmHash.
+1. Optionally, add tags you want to associate with the matched events.
+1. Add a name for the scanning rule.
+1. Optionally, add a description for the rule.
+1. Click **Add Rule**.
+
+##### Path notation example {#path-notation-example-custom}
+
+ For the following message structure, use `outer_key.inner_key.double_inner_key` to refer to the key with the value `double_inner_value`.
+```json
+{
+    "outer_key": {
+        "inner_key": "inner_value",
+            "a": {
+                    "double_inner_key": "double_inner_value",
+                    "b": "b value"
+                },
+            "c": "c value"
+        },
+        "d": "d value"
+    }
```

Comment:
Not that it matters much for an example like this, but it seems like a closing bracket is missing at the bottom here. The indents are a bit strange as well, "a" should line up with "inner_key" for example.

Reply:
This is used in multiple places so will fix all in on commit.

Reply:
Fixed